packages/react/counter/src/index.ts

```diff
@@ -1,2 +1,11 @@
 export { Container as Counter } from './Container'
-export { RelativeTimeRange } from '@propeldata/ui-kit-graphql'
+
+export {
+  type FilterInput,
+  FilterOperator,
+  type InputMaybe,
+  Propeller,
+  RelativeTimeRange,
+  type Scalars,
+  type TimeRangeInput
+} from '@propeldata/ui-kit-graphql'
```

Comment:
and exporting just everything?

Reply:
It will result in a bunch of irrelevant stuff, at least for now, while we have split packages.

Once we combine everything into a single library, then yes, we should do this 👍